packages/consts/src/consts.ts

```diff
@@ -376,6 +376,7 @@ export const ENV_VARS = {
     FACT: 'APIFY_FACT',
     DEDICATED_CPUS: 'APIFY_DEDICATED_CPUS',
     SDK_LATEST_VERSION: 'APIFY_SDK_LATEST_VERSION',
+    INTERNAL_PORT: 'APIFY_INTERNAL_PORT',
```

Comment:
I have absolutely no idea where this is coming from. Why add it?

Reply:
it is used just few lines below this, we need to define all constats that are used in the code, otherwise TS will complain. 

(or we could typecast, which is rather ugly in this case, as we'd have to do `(ENV_VARS as any).INTERNAL_PORT`)

Reply:
actually looking at the original code, and apparently this was never working, as the `ENV_VARS.INTERNAL_PORT` was not defined, so the usage in `INTEGER_ENV_VARS` resulted in `undefined`

so what to do with this, remove both usages I guess?

Reply:
Yeah, I think we can delete it. I've never seen it used anywhere.